python/version_details/version_details.py

```diff
@@ -586,7 +587,7 @@ def load_data(self, entity):
         self._requested_entity = entity
 
         # If we're pinned, then we don't allow loading new entities.
-        if self._pinned and self.current_entity:
+        if self._pinned and self._current_entity:
```

Comment:
This overall concept of the change looks good to me. I have a few concerns though:
1. Need to make sure that wherever the code was switched to use `self.current_entity`, that the code isn't dependent on some other data that doesn't exist yet until `self._current_entity` is set.
2. Need to go through other tk components that reference `version_details_panel.current_entity` and do the same - make sure that they aren't relying on some other data that is not available yet when `._request_entity` is set.
3. Need to go through and cleanup other tk components that have checks for `or panel._requested_entity` since I think I added several of those during the sg3dr work.